cincinnati/src/plugins/internal/graph_builder/dkrv2_openshift_secondary_metadata_scraper/test_fixtures/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+set -xu
+export REPO_BASE="registry.svc.ci.openshift.org/cincinnati-ci-public/cincinnati"
+export AUTHFILE="${CINCINNATI_CI_PUBLIC_DOCKERJSON_PATH}"
+../../../../../../../graph-builder/tests/images/build-n-push-buildah.sh graph-data-6420f7fbf3724e1e5e329ae8d1e2985973f60c14
```

Comment:
Where is this build.sh is used?

Reply:
It's not. We haven't fully moved to registry CI (see #308) and we therefore lack automation to build and push these images on CI. I didn't want to pull that whole topic into this PR.

Reply:
GTo complete the picture: I ran this script manually from my developer machine to build and push the test image that's currently referenced [here](https://github.com/openshift/cincinnati/pull/322/files#diff-f9cab18d8b1f8be7bc1faf5d75628668fdc8289297baaeed591b26d15fc87ecdR352).